src/service/growthService.js

```
@@ -0,0 +1,13 @@
+import axiosInstance from "./axiosInstance";
+
+class GrowthService {
+  static async getDob() {
+    try {
+      return axiosInstance.get("/growth/dob");
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)